graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js

```diff
@@ -2,7 +2,7 @@ import Reflux from 'reflux';
 
 import URLUtils from 'util/URLUtils';
 import ApiRoutes from 'routing/ApiRoutes';
-import fetch from 'logic/rest/FetchProvider';
+import {fetch, fetchPeriodically} from 'logic/rest/FetchProvider';
```

Comment:
In this case this is also not the recommended way of importing both functions (it breaks the [`import/named`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md) eslint rule). We should use this instead:
`import fetch, { fetchPeriodically } from 'logic/rest/FetchProvider';`